src/components/material/material-buttons/helper.ts

```diff
@@ -29,4 +29,13 @@ export const isArticle = (manifestation: Manifestation) => {
   );
 };
 
+export const hasCorrectMaterialType = (
+  desiredMaterialType: string,
+  allMaterialTypes: Manifestation["materialTypes"]
```

Comment:
Please change the argument type (and name) to `Manifestation` instead of `Manifestation["materialTypes"]`.

In my opinion this has three advantages:

1. It reads better: We check whether the manifestation has the correct material type "whatever". The location of the material types within the manifestation is an implementation detail.
2. There is less typing for the caller
3. It follows the pattern of the other `hasCorrect` functions in this file.

Consider refactoring `isArticle` to call `hasCorrectMaterialType("tidsskriftsartikel", manifestation), ||  hasCorrectMaterialType("avisartikel", manifestation)`.